File: Swift-C++-Testsuite/test/controlFlow/C++/SwitchFallthrough.cpp

int switchMe(int input){
    int counter = 0;

    switch (input) {
        case 2: 
        counter = 100;
        case 3: 
        counter -= 23;
        default:
        counter = 42;
    }
    return counter;
}

int main(){
    int res = switchMe(2);
    return 0;
}

File: Swift-C++-Testsuite/test/controlFlow/Swift/SwitchFallthrough.swift
@main
struct MyMain {

    static func main() {
        let res = switchMe(input: (2))
    }

    static func switchMe(input: Int) -> Int {
        var counter = 0
        switch input {
        case 2:
            counter = 100
            fallthrough
        case 3:
            counter -= 23
        default:
            counter = 42
        }
        return counter
    }
}
